src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1701,7 +1701,7 @@ public Allele getAltAlleleWithHighestAlleleCount() {
         return getAlternateAlleles().stream()
                 .map(allele -> new Tuple<>(allele, getCalledChrCount(allele)))
                 .max((alleleAndCount1, alleleAndCount2) -> Integer.compare(alleleAndCount1.b, alleleAndCount2.b))
-                .get()
+                .orElse(null)
                 .a;
```

Comment:
I think something like this may be a more direct way to get the value you want:

```
     return getAlternateAlleles().stream()
                .max(Comparator.comparing(this::getCalledChrCount))
                .orElse(null);
```